homeassistant/components/airvisual/sensor.py

```diff
@@ -185,18 +185,14 @@ def __init__(self, coordinator, config_entry, kind, name, icon, unit, locale):
         self._kind = kind
         self._locale = locale
 
-    @property
-    def available(self) -> bool:
-        """Return if entity is available."""
-        return super().available and self.coordinator.data["current"]["pollution"]
-
     @callback
     def update_from_latest_data(self):
         """Update the entity from the latest data."""
         try:
             data = self.coordinator.data["current"]["pollution"]
         except KeyError:
             return
+        self._attr_available = super().available and data
```

Comment:
This won't have effect, as `available` is an overridden method already.

Reply:
How should this look?

Reply:
By implementing the available property method. The available property method is overridden by the CoordinatorEntity that this class implements, rendering `_attr_available` to be unused.